### DISCUSSION TITLE
Do not show Keepass XC screen when screen sharing on zoom calls

### DISCUSSION THREAD

**Discussion by kishaningithub** [keepassxreboot/keepassxc#9905] [Ideas]
This app has a great security feature to not allow screen capturing when taking screenshots(screenshot below), can the implementation of this be updated to also make this app not visible during zoom calls / other screen sharing apps.

<img width="239" alt="image" src="https://github.com/keepassxreboot/keepassxc/assets/763760/1af4296c-122b-4597-91aa-3d03e315a329">

PS - I am using a Mac

**Comment by droidmonkey** [comment]
It should already be hidden from screen share, if not then that is a Mac problem. We tell the operating system to hide our window, it decides how and when to do that.

**Reply by kishaningithub** [reply]
Thanks for the info @droidmonkey !

Can anyone using Mac kindly let know how you are configuring mac to prevent this?
